util/lua/lua.go

```diff
@@ -333,6 +349,29 @@ func GetConfigMapKey(gvk schema.GroupVersionKind) string {
 	return fmt.Sprintf("%s/%s", gvk.Group, gvk.Kind)
 }
 
+func GetWildcardConfigMapKey(vm VM, gvk schema.GroupVersionKind) (string, error) {
+	var gvkKeyToMatch string
+
+	if gvk.Group == "" {
+		gvkKeyToMatch = gvk.Kind
+	} else {
+		gvkKeyToMatch = fmt.Sprintf("%s/%s", gvk.Group, gvk.Kind)
+	}
```

Comment:
```suggestion
	gvkKeyToMatch := GetGroupWildcardConfigMapKey(gvk)
```

Reply:
Oh the GetGroupWildcardConfigMapKey was garbage left from initial attempts.
Deleted it. 
Removed code duplication using GetConfigMapKey.